reference/tax_settings.v3.yml

```
@@ -178,6 +190,18 @@ components:
           default: true
           type: boolean
           description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: When enabled, geolocation data will be used to match the guest shopper with a relevant country-level tax zone for tax estimation on the storefront. The geolocation data will be used to determine the corresponding tax zone provided in basic tax - once the corresponding tax zone is found, the tax estimation will be applied to the store.
+        guest_shopper_tax_zone_id:
+          default: 1
+          type: integer
+          description: The setting allows the user to configure the guest shopper tax zone for shoppers to see in the storefront. The user can configure the tax zone id to any existing tax zone in basic tax where the chosen tax zone will be used for tax estimation. When used in conjunction with store_tax_zone_id and should_subtract_store_tax, the rules of should_subtract_store_tax_will apply for the storefront where the tax rates will be subtracted and recalculated based on the two tax zones provided.
+        store_tax_zone_id:
+          default: 1
+          type: integer
+          description: This setting applies only if a merchant enters tax-inclusive prices and has should_subtract_store_tax enabled. This setting lets the user to configure their own store tax zone in basic tax. When the shopper enters a destination address or provides their own geolocation, the rules of should_subtract_store_tax will apply with regards to tax estimation. This correlates to the store tax rate description provided in the should_subtract_store_tax where the initial tax rates are calculated in the tax zone id provided in this configuration.
```

```suggestion
          description: This setting applies only if a merchant enters tax-inclusive prices. Describes which tax zone to use for store tax subtraction.
```

I think we can avoid some detail here/leave it for other sections 🙏

---

updated 👍